rets/http/client.py

```diff
@@ -35,7 +36,7 @@ def __init__(self,
 
         splits = urlsplit(login_url)
         self._base_url = urlunsplit((splits.scheme, splits.netloc, '', '', ''))
-        self._capabilities = {
+        self._capabilities = capability_urls or {
```

Comment:
I think we could remove `capability_urls` since it's encapsulated by `login_url`.